### ISSUE TITLE
Email Notifications

### ISSUE THREAD

**Issue by brandonsamuel1** [brandonsamuel1/Purrfect-Match#1]
**Task 1:** When users register, they will provide their email which will be used to send email notification when they adopt a feline friend! - COMPLETED

**Task 2:** Add "Adopt" button on the show page for a single feline - COMPLETED

**Task 3:** Once user clicks on "Adopt" button, display a modal that will provide and description of the feline they are about to adopt, and a "Proceed" and  "Cancel" button - COMPLETED

**Task 4:** If user clicks "Proceed", with the email registered under their account, send a confirmation/thank you email for their new adoption. Otherwise, "Cancel" button will take user back to the show page - COMPLETED(However, going to spam folder)

**Task 5:** Once adoption has been processed, update database with boolean set to true. Then hide all listings with boolean "true" from the home page so no one can view the listing. - COMPLETED

**Comment by cpb** [comment]
Yup, but just for clarification "then hide" will actually be the default -- the index page will not show felines that have been adopted. But from this a linear perspective, yes, that feline will disappear from the listing.

**Comment by brandonsamuel1** [comment]
Gotcha! Thanks
